opsmngr/version_manifest_test.go

```diff
@@ -46,7 +46,10 @@ func TestVersionManifest_Get(t *testing.T) {
 				  "architecture": "amd64",
 				  "gitVersion": "1c1c76aeca21c5983dc178920f5052c298db616c",
 				  "platform": "osx",
-				  "url": "/osx/mongodb-osx-x86_64-2.6.0.tgz"
+				  "url": "/osx/mongodb-osx-x86_64-2.6.0.tgz",
```

Comment:
I am not sure why github shows the line in this way. In my Intellj it looks okay
<img width="688" alt="Screenshot 2022-02-16 at 17 28 28" src="https://user-images.githubusercontent.com/5663078/154321793-db6a11b9-603d-486f-a626-002521acc92e.png">

Reply:
tabs vs spaces, did you use tabs or spaces?

Reply:
Tab 
https://media.giphy.com/media/l0IylSajlbPRFxH8Y/giphy.gif